core/runtime/binaryen/wasm_memory_impl.cpp

```diff
@@ -177,4 +177,13 @@ namespace kagome::runtime::binaryen {
     }
   }
 
+  PointerSize WasmMemoryImpl::storeBuffer(const common::Buffer &value) {
```

Comment:
Think about make template function for using that with different flat containers

Reply:
refactored both storeBuffer methods to take gsl::span<const uint8_t> instead of buffer. It required to add toSpan() and toConstSpan() methods to Buffer, since in tests mocking implicit type casts don't work and exact types are required.